docs/azure_arc_jumpstart/azure_arc_k8s/day2/aks/aks_policy_calico/_index.md

```diff
@@ -0,0 +1,124 @@
+---
+type: docs
+title: "Use the Azure Policy Add-on for AKS with a custom Azure Policy to verify whether Calico network policy is enabled on the AKS cluster"
+linkTitle: "Use the Azure Policy Add-on for AKS with a custom Azure Policy to verify whether Calico network policy is enabled on the AKS cluster"
```

Comment:
The title is too long

Reply:
chaneg to:
title: "Use the Azure Policy Add-on to audit Calicocloud / calico enterprise"
linkTitle: "Use the Azure Policy Add-on to create a custom Azure Policy for auditing calicocloud/calico enterprise components"